### PR TITLE
add avoid-escape flag to quotes rule

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@jridgewell PTAL

File: extensions/amp-font/0.1/test/test-fontloader.js [hunk 1]
```diff
@@ -22,19 +22,19 @@ import * as sinon from 'sinon';
 adopt(window);
 
 /** @private @const {string} */
-const FONT_FACE_ = '\
```
Comment (jridgewell) [inline extensions/amp-font/0.1/test/test-fontloader.js]: Back ticks are multi-line without these backslashes.

Comment (erwinmombay) [inline extensions/amp-font/0.1/test/test-fontloader.js]: yeah, didnt make the change here since i just used `lint --fix`. you want me to make the change?

Comment (jridgewell) [inline extensions/amp-font/0.1/test/test-fontloader.js]: Yup.

Comment (erwinmombay) [inline extensions/amp-font/0.1/test/test-fontloader.js]: done.

**Comment by jridgewell** [comment]
LGTM.

**Pull request merged**